lib/OpenQA/Utils.pm

```diff
@@ -1030,7 +1030,8 @@ sub mark_job_linked {
         my $found    = 0;
         my $comments = $job->comments;
         while (my $comment = $comments->next) {
-            if ($comment->label eq 'linked') {
+            my $label = $comment->label;
```

Comment:
I personally find `($comment->label || '') eq 'linked'` good enough and less noisy in general - but just a nitpick

Reply:
In that case defined or (`($comment->label // '') eq 'linked'`) would be a little more correct.